src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -189,6 +189,26 @@ private void CheckTrainingParameters(ImageClassificationEstimator.Options option
             return (jpegData, resizedImage);
         }
 
+        private static Tensor Encode(VBuffer<byte> buffer)
+        {
+            int length = buffer.Length;
+            var size = c_api.TF_StringEncodedSize((UIntPtr)length);
+            var handle = c_api.TF_AllocateTensor(TF_DataType.TF_STRING, IntPtr.Zero, 0, (UIntPtr)((ulong)size + 8));
+            //AllocationType = AllocationType.Tensorflow;
```

Comment:
>//AllocationType = AllocationType.Tensorflow; [](start = 12, length = 45)

remove and please verify what AllocationType in TF .NET means since we are not setting it to any value.

Reply:
This is not really used. It was from another code snippet in Tensorflow .net